### ISSUE TITLE
Resistor mode, etc non-functional

### ISSUE THREAD

**Issue by ctacke** [WildernessLabs/Meadow.Linux#1]
M4L currently only supports the sysfs driver.  Need to implement the character driver to unlock these.

**Comment by microhobby** [comment]
what are the plans here? use libgpiod interop for this (short path), or implement the interop ioctl calls?

**Comment by ctacke** [comment]
Yes, sysfs was really just a quick path to prove out it all works.  Plan is to go straight to ioctls, which matches how the I2C and SPI work.
